examples/jsm/nodes/materials/NodeMaterial.js

```diff
@@ -32,9 +32,12 @@ class NodeMaterial extends ShaderMaterial {
 
 		this.forceSinglePass = false;
 
+		this.unlit = this.constructor === NodeMaterial.prototype.constructor; // Extended materials are not unlit by default
+
+		this.fog = true;
 		this.lights = true;
 		this.normals = true;
-		this.unlit = this.constructor === NodeMaterial.prototype.constructor; // Extended materials are not unlit by default
+		this.colorSpace = true;
```

Comment:
Maybe `applyColorSpace`?

Reply:
Maybe we can make it more similar with `material.toneMapped`?

https://github.com/mrdoob/three.js/blob/ce0b3d0415c4bc87a8b4ad01c5d21cc26c6ec5eb/src/materials/Material.js#L71-L73

Reply:
So... `colorSpaced`?